WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -516,23 +525,21 @@ private extension DashboardViewModel {
                                                 canShowLastOrders: canShowLastOrders,
                                                 canShowInbox: canShowInbox)
 
-        // Next, get saved cards and preserve existing enabled state for all available cards.
-        // This is needed because even if a user already disabled an available card and saved it, in `initialCards`
-        // the same card might be set to be enabled. To respect user's setting, we need to check the saved state and re-apply it.
-        let savedCards = await loadDashboardCards() ?? []
-        let updatedCards = initialCards.map { initialCard in
-            if let savedCard = savedCards.first(where: { $0.type == initialCard.type }),
-               savedCard.availability == .show && initialCard.availability == .show {
-                return initialCard.copy(enabled: savedCard.enabled)
-            } else {
-                return initialCard
-            }
-        }
-
         /// If no saved cards are found, display the default cards.
         if savedCards.isEmpty {
-            dashboardCards = updatedCards
+            dashboardCards = initialCards
         } else {
+            // Based on saved cards, preserve existing enabled state for all available cards.
+            // This is needed because even if a user already disabled an available card and saved it, in `initialCards`
+            // the same card might be set to be enabled. To respect user's setting, we need to check the saved state and re-apply it.
+            let updatedCards = initialCards.map { initialCard in
+                if let savedCard = savedCards.first(where: { $0.type == initialCard.type }),
+                   savedCard.availability == .show && initialCard.availability == .show {
+                    return initialCard.copy(enabled: savedCard.enabled)
+                } else {
+                    return initialCard
+                }
+            }
```

Comment:
This might seem like a big change but it's essentially just updating the logic to no longer load saved cards anymore,  the rest is the same.